---
title: AEP - Não é possível conectar um ao Big Query como uma fonte de dados ao adicionar dados
description: Descrição
solution: Experience Platform
product: Experience Platform
applies-to: Experience Platform
keywords: KCS, AEP, fontes, Google, Consulta grande, conjuntos de dados
resolution: Resolution
internal-notes: null
bug: false
article-created-by: Christophe Jossic
article-created-date: 5/13/2022 2:36:35 PM
article-published-by: Christophe Jossic
article-published-date: 5/13/2022 2:39:23 PM
version-number: 2
article-number: KA-19525
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=1a607b16-cad2-ec11-a7b5-00224809c27a
exl-id: 14017ad5-9ac9-4932-a596-fabaf45ab773
source-git-commit: 0c3e421beca46d9fe1952b1f98538a50697216a0
workflow-type: tm+mt
source-wordcount: '165'
ht-degree: 1%

---

# AEP - Não é possível conectar um ao Big Query como uma fonte de dados ao adicionar dados

## Descrição


Estamos tentando conectar nossa instância de Produção a uma tabela de Consulta Grande e recebemos um erro -<b>*&quot;Ocorreu um erro ao obter dados da origem&quot;</b>*

Os fluxos de dados existentes estão em execução com êxito.

Adicionar novos dados parece ser o problema.


## Resolução


O problema está no <b>[!DNL Microsoft] API do DataFactory </b>O que leva muito tempo para buscar a lista de tabelas se houver um grande número de conjuntos de dados no projeto GBQ. A correção para isso foi feita por [!DNL Microsoft] mas parece que eles o lançaram somente para clientes afetados no momento do relatório. A engenharia da Adobe está se comunicando com o [!DNL Microsoft] para que a correção seja lançada para todos. Se estiver enfrentando esse problema da perspectiva do Adobe Experience Platform, por favor, crie um tíquete de suporte para a equipe da AEP.
